tools/public_api_guard/cdk/testing.d.ts

```diff
@@ -115,6 +122,12 @@ export interface ModifierKeys {
     shift?: boolean;
 }
 
+export declare function noAutoChangeDetection<T>(fn: () => Promise<T>): Promise<T>;
+
+export declare function parallel<T>(values: Iterable<T | PromiseLike<T>>): Promise<T[]>;
+
+export declare function stopHandlingChangeDetectionBatching(): void;
```

Comment:
Should this be documented for environment authors?

Reply:
Added docs for this